integration-cli/docker_cli_daemon_test.go

```diff
@@ -564,6 +564,42 @@ func (s *DockerDaemonSuite) TestDaemonBridgeIP(c *check.C) {
 	deleteInterface(c, defaultNetworkBridge)
 }
 
+func (s *DockerDaemonSuite) TestDaemonRestartWithBridgeIPChange(c *check.C) {
+
+	if err := s.d.Start(); err != nil {
```

Comment:
You should call `defer d.Restart()` after to be sure, that all args influence will be dropped.